### DISCUSSION TITLE
Hydrogen production footprint?

### DISCUSSION THREAD

**Discussion by thomasgibon** [electricitymaps/electricitymaps-contrib#4812] [Ideas]
**TL;DR**: It would be quite easy to add a "hydrogen production" layer to the existing electricity map. We would observe that production of hydrogen grid electrolysis is actually a bad idea in many places in the world – even though this hydrogen could be labeled as "green".

The European Commission is currently debating on the set of rules that will dictate hydrogen production certification. For those familiar with RECs, the same principles will probably apply to hydrogen: [no additionality, no time-matching](https://www.euractiv.com/section/energy/news/revealed-how-germany-stepped-in-to-delay-eu-green-hydrogen-rules/).

[Google has recently stepped up](https://www.euractiv.com/section/energy/news/google-wants-more-restrictive-green-hydrogen-rules/) against the current prospect of such a permissive certification framework, claiming that 24/7 certification should apply to hydrogen.

Now, electrolysis requires about 55-60 kWh/kg H2 (depends on the actual method and if you include steps like compression), so it would be quite easy to add a "hydrogen" layer to the electricity map, in kg CO2 eq./kg H2 produced in each country, simply by multiplying the live grid emission factors by 55-60. We would certainly observe that the thresholds of 3 kg CO2 eq. (as noted in the EU Taxonomy) or even 10 kg CO2 eq. (grey hydrogen) are almost always exceeded.

Here's a figure to illustrate the problem, from about 200 g CO2 eq./kWh, then electrolysis is worse than the conventional way of making hydrogen from natural gas.

![image](https://user-images.githubusercontent.com/23332955/204254692-01ae70e6-3c85-4921-a09b-54b2bd318a0d.png)

**Technical note**: I actually think the mis-certification risk is even higher than with electricity, why?

Because of how efficient electric systems are, it is quite rare to find an electric alternative that will emit more CO2 than its fossil counterpart. Even an electric car charged with coal electricity will emit more or less the same as a diesel car, but probably not much more. On the other hand, if you electrolyse water with coal power, you will emit 50-60 kg CO2/kg of hydrogen, when steam methane reforming (the conventional "dirty" way) would emit about 10 kg CO2/kg. Because of this, I personally think the greenwashing risk is even *much* higher with hydrogen than with electricity.

**Comment by VIKTORVAV99** [comment]
It's definitely an interesting idea and could even help with getting more accurate electricity emission factors down the road if hydrogen is used as long-term storage for electricity generation (already planned in some places.

However, this is something the data team would have to look closely at, and they are quite busy at the moment so it might take some time. I think we might also need to find H2 cross border pipeline and generation information (with live data) if we want to show it as an accurate representation on the map (both production and consumption like the electricity map).

**Reply by thomasgibon** [reply]
Thanks – it's definitely a long-term idea, I know how busy the team is, but I'm working on the topic at the moment so I wanted to drop the idea "for future use". Producing hydrogen with coal power is non-sensical, but it will be allowed in Europe, I just wanted to share my frustration here :)

Pipeline information would be great to have for sure, as H2 will probably be blended, and gas traceability is already not great, this may cause a few methodological headaches...
